deploy/charts/origin-ca-issuer/templates/_helpers.tpl

```diff
@@ -0,0 +1,41 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "origin-ca-issuer.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
```

Comment:
You append to this in some places (e.g. `-controller`), so you need to need to account for that here

Reply:
Don't tell [cert-manager](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/jetstack/cert-manager%24%40master+cert-manager.fullname&patternType=literal).